.github/CODEOWNERS

```diff
@@ -1038,9 +1038,22 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 /x-pack/plugins/entity_manager @elastic/obs-entities
 /x-pack/test/api_integration/apis/entity_manager @elastic/obs-entities
 
+
 # Data Discovery
-/test/plugin_functional/plugins/data_search @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/alias @elastic/kibana-data-discovery
+/test/functional/page_objects/context_page.ts @elastic/kibana-data-discovery
+/test/functional/services/data_views.ts @elastic/kibana-data-discovery
+/test/functional/services/saved_objects_finder.ts @elastic/kibana-data-discovery
+/test/functional/page_objects/unified_search_page.ts @elastic/kibana-data-discovery
```

Comment:
```suggestion
```
Just one more left to drop here. This one is a similar situation as `/test/api_integration/apis/suggestions` and is owned by @elastic/kibana-visualizations.